param/__init__.py

```diff
@@ -2040,3 +2040,16 @@ def _validate(self, val):
         # values are numeric, which is redundant, so just call
         # _checkBounds().
         self._checkBounds(val)
+
+
+class Trigger(Boolean):
+
+    @instance_descriptor
+    def __set__(self,obj,val):
```

Comment:
Nitpicky but I'd like param to eventually have a consistent style again.

```suggestion
    def __set__(self, obj, val):
```

Reply:
Resolved, but not applied?